compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -87,6 +87,7 @@ interface PlatformContext {
         fun onRootForTestDisposed(root: PlatformRootForTest)
     }
 
+    // TODO: discuss whether this interface should be nested in PlatformContext
```

Comment:
What's wrong with the current approach?

cc @igordmn

Reply:
It's not consistent with the other codebase. We never nest interfaces, that are used somewhere else or have a descriptive name. `PlatformContext.SemanticsOwnerListener` is not private and it presumes, that there are other sorts of `SemanticOwnerListener` related to something else.